### DISCUSSION TITLE
Breaking changes in latest version of flutter, which is affecting riverpod performance

### DISCUSSION THREAD

**Discussion by devmarwat** [rrousselGit/riverpod#2575] [General]
As the issue needs to be discussed in more detail so I record a video on to address it properly.
https://youtu.be/lRtrcG_Sz8U

**Comment by hamza-imran75** [comment]
Could you please provide reproducable code, maybe use Zapp if possible.

**Comment by rrousselGit** [comment]
Please make a proper issue and make a reproducible sample

**Reply by devmarwat** [reply]
My code is lengthy and  I don't know how to explain the issue that why I record a video on it. If you want then I can upload the whole application code to GitHub and from there you test the whole application.

**Reply by hamza-imran75** [reply]
Could you please isolate the error, and create a minimum reproducable code.

**Reply by devmarwat** [reply]
I don't know how would I do that.

**Comment by devmarwat** [comment]
Here is the full code of my application. 
https://github.com/Hassanmrwt/Flutter-Realtime-messaging-app-build-with-appwrite-and-riverpod
